snippets/base/migrations/0062_set_asrsnippet_id_autoincrement_starting_point.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.1.3 on 2018-11-16 12:30
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('base', '0061_auto_20181116_0810'),
+    ]
+
+    operations = [
+        migrations.RunSQL(['ALTER TABLE base_asrsnippet AUTO_INCREMENT=10500;'], [''])
```

Comment:
Why is this necessary?

Reply:
I want Snippets and ASRSnippets to have the same IDs for metrics. To avoid ID conflicts with new ASRSnippets I move the auto-increment point beyond the current Snippets point (which is around 9800 now)

Reply:
Ah! Makes sense. This ensures no conflicts with the ported snippets. 👍